lib/matplotlib/tests/test_figure.py

```diff
@@ -796,7 +796,6 @@ def test_clf_not_refedined():
         # check that subclasses do not get redefined in our Figure subclasses
         assert 'clf' not in klass.__dict__
 
-
```

Comment:
Hm... Is flake8 not getting run on tests? I'm pretty sure this line should be there. This file should also be left alone if you aren't touching it anymore.

Reply:
Fixed